src/lib/containers/entity_finder/details/view/DetailsView.tsx

```diff
@@ -450,21 +449,10 @@ export const DetailsView: React.FunctionComponent<DetailsViewProps> = ({
                   return (
                     <>
                       Version
-                      <MarkdownPopover
```

Comment:
Used the DetailsView as the basis for the initial implementation for HelpPopover